README.md

```diff
@@ -1 +1,15 @@
-Read me goes here!
+# README
+
+## Before you start
+
+1. Clone the respository to the file/folder of your choice.
+2. Then navigate to the `tic_tac_toe` folder in your terminal and run `bundle install`.
+
```

Comment:
What happens if I don't have Ruby installed or the required version of Ruby?

Reply:
- Added a link to the Ruby installation instructions on ruby-lang.org. Also specified the version was 2.6.3